src/mono/System.Private.CoreLib/src/System/Reflection/CustomAttributeData.cs

```diff
@@ -127,25 +127,77 @@ IList<CustomAttributeNamedArgument> NamedArguments
 
         public static IList<CustomAttributeData> GetCustomAttributes(Assembly target)
         {
-            return CustomAttribute.GetCustomAttributesData(target);
+            if (target is null)
+                throw new ArgumentNullException(nameof(target));
+
+            return target.GetCustomAttributesData();
         }
 
         public static IList<CustomAttributeData> GetCustomAttributes(MemberInfo target)
         {
-            return CustomAttribute.GetCustomAttributesData(target);
+            if (target is null)
+                throw new ArgumentNullException(nameof(target));
+
+            return target.GetCustomAttributesData();
+        }
+
+        public static IList<CustomAttributeData> GetCustomAttributes(Module target)
+        {
+            if (target is null)
+                throw new ArgumentNullException(nameof(target));
+
+            return target.GetCustomAttributesData();
+        }
+
+        public static IList<CustomAttributeData> GetCustomAttributes(ParameterInfo target)
+        {
+            if (target is null)
+                throw new ArgumentNullException(nameof(target));
+
+            return target.GetCustomAttributesData();
         }
```

Comment:
Please move this code to the shared location. It's should be pretty simple and make the change clearer.

Reply:
OK, will do.